specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/examples/documents.json

```diff
@@ -6,8 +6,16 @@
     "endpoint": "{endpoint}",
     "Ocp-Apim-Subscription-Key": "{API key}",
     "id": "727BF148-F327-47A0-9481-ABAE6362F11E",
-    "$top": 5,
-    "$skip": 10
+    "$top": 20,
+    "$skip": 10,
+    "$maxpagesize": 10,
```

Comment:
given this new parameters, looks like sample needs to be updated so it returns 10 documents and no `@nextlink`